crates/cdk-sqlite/src/wallet/mod.rs

```diff
@@ -781,6 +783,7 @@ fn sqlite_row_to_mint_info(row: &SqliteRow) -> Result<MintInfo, Error> {
             .unwrap_or_default(),
         mint_icon_url,
         motd,
+        time: Some(unix_time()),
```

Comment:
Adding it here won't tell us what the unix time the Mint last told us it was it will tell us what the unix time currently is for the wallet.

Reply:
addressed in 347b8a19caf955f87174e3b389e3069794875078